benchmark/torch/ppo/README.md

```diff
@@ -0,0 +1,34 @@
+## Reproduce PPO with PARL
+Based on PARL, the PPO algorithm of deep reinforcement learning has been reproduced, reaching the same level of indicators as the paper in mujoco benchmarks.
+
+Include following approach:
++ Clipped Surrogate Objective
++ Adaptive KL Penalty Coefficient
```

Comment:
please remove line 4~6,  torch based ppo uses the "clipped" method, and does not provide two approachs